src/WebJobs.Script/Workers/Rpc/IRpcWorkerConcurrencyChannel.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Workers.Rpc
+{
+    internal interface IRpcWorkerConcurrencyChannel
```

Comment:
I don't think you need this new interface at all. Channels already implement GetWorkerStatusAsync and we should be able to return everything we need via WorkerStatus.

Reply:
Removed the interface.